src/open_dread_rando/door_locks/custom_door_types.py

```diff
@@ -354,19 +352,20 @@ class BaseShield:
     def __init__(self, shield: ShieldData):
         self.data = shield
 
-    def patch_model_data(self, new_template: dict, editor: PatcherEditor, version: str):
```

Comment:
rename new_template to `actordef` or something since it's not really using templates anymore